### DISCUSSION TITLE
Is docker no longer a supported method?

### DISCUSSION THREAD

**Discussion by yourjelly** [hlky/stable-diffusion-webui#1446] [General]
The `.env_docker` and `docker-compose.yml` files no longer exist in the repo

**Comment by yourjelly** [comment]
https://sd-webui.github.io/stable-diffusion-webui/docs/3.docker-guide.html

The guide still exists but is not possible.

**Comment by codedealer** [comment]
It is in the process of an overhaul. 
You can use prebuilt images to run Stable Diffusion Web UI via Docker.

https://hub.docker.com/r/hlky/sd-webui

**Reply by yourjelly** [reply]
Thanks, I was looking for that but wasn't coming up on google.
